src/main.py

```diff
@@ -524,31 +532,22 @@ def loop():
 				player.play_speech(resources_path + 'alexayes.mp3')
 				time.sleep(0.5)
 
-		# do the following things if either the button has been pressed or the trigger word has been said
-		if debug:
-			print("detected the edge, setting up audio")
-
 		# To avoid overflows close the microphone connection
 		inp.close()
 
+		silence_listener(VAD_THROWAWAY_FRAMES)
+		alexa_speech_recognizer()
```

Comment:
Nice work putting this before the speech recognition. I had a note in my file to move it also.